README.md

```diff
@@ -136,13 +136,13 @@ In this example we register a custom font "MyFont" with sans fallback `MyFont, s
     (weight: bold, style: normal)
 )) {
     @font-face {
-        src: url('MyFont-Thin.woff');
+        src: url('MyFont-Thin.woff2'), url('MyFont-Thin.woff');
```

Comment:
does the example need updating with `format`?

Reply:
Done